lib/install/management/appliance.go

```diff
@@ -605,6 +606,57 @@ func (d *Dispatcher) createAppliance(conf *config.VirtualContainerHostConfigSpec
 	},
 	)
 
+	// Kubelet
+	if conf.KubernetesServiceURL != "" && conf.KubeletConfigFile != "" {
+		vmName, err := vm2.Name(d.op)
+		if err != nil {
+			d.op.Errorf("Failed to get VM name, error: %s", err)
+			return err
+		}
+		kubeletName := fmt.Sprintf("kubelet-%s", vmName)
+		kubeletStarter := executor.Cmd{
+			Path: "/sbin/kubelet-starter",
+			Args: []string{
+				"/sbin/kubelet-starter",
+			},
+		}
+
+		kubeletStarter.Env = append(kubeletStarter.Env, fmt.Sprintf("%s=%s", "KUBELET_NAME", kubeletName))
+
+		plPort := portLayerPort
+		if conf.Diagnostics.DebugLevel > 2 {
+			// expose portlayer service on client interface
+			plPort = constants.DebugPortLayerPort
```

Comment:
In the other places `constants.DebugPortLayerPort` is used, it's used from the analog of `kubelet-starter/main.go`. Is there a reason we're not following that same pattern here?